### ISSUE TITLE
Singletons in output

### ISSUE THREAD

**Issue by benjeffery** [benjeffery/tsinfer-snakemake#41]
We're seeing singletons in the output tree sequences when they should be fully filtered. I'm not yet sure if this is a pipeline issue or a tsinfer issue.

**Comment by benjeffery** [comment]
![Screenshot from 2024-04-25 12-32-46](https://github.com/benjeffery/tsinfer-snakemake/assets/8552/7a3a98e6-8505-4a12-b4a3-8f8ed9f50d26)

From the tsinfer sampledata, this site is a doubleton, but is a singleton in the resulting tree sequence.

**Comment by hyanwong** [comment]
> From the tsinfer sampledata, this site is a doubleton, but is a singleton in the resulting tree sequence.

Hmm: that means we aren't losslessly storing the genotypes, right?

**Comment by benjeffery** [comment]
> > From the tsinfer sampledata, this site is a doubleton, but is a singleton in the resulting tree sequence.
> 
> Hmm: that means we aren't losslessly storing the genotypes, right?

Yes, have found some issues with the sample masking - trying to square it all off.
